package.json

```diff
@@ -40,7 +40,7 @@
     "bootstrapping"
   ],
   "bin": {
-    "create-pika-app": "dist-node/index.bin.js"
+    "pika-init": "dist-node/index.bin.js"
```

Comment:
Did you still need this here? The  `"@pika/plugin-simple-bin"` plugin should add this for you automatically

Reply:
No, I thought it was a requirement to spit out the correct .bin.js, do whatever you want with it!

Reply:
np, I'll confirm when I publish and remove